packages/datagrid/src/components/DefaultCellRenderer/DefaultValueRenderer.component.js

```diff
@@ -39,7 +39,14 @@ export default class DefaultValueRenderer extends React.Component {
 	}
 
 	render() {
-		const stringValue = String(this.props.value);
+		let stringValue;
+
+		if (this.props.value === null) {
```

Comment:
How about
```suggestion
		if (!this.props.value) {
```
To handle weird cases like undefined ?